lib/menu.rb

```diff
@@ -0,0 +1,35 @@
+class Menu
+
+  attr_reader :dishes_available
+
+  def initialize
+    @dishes_available = {}
+  end
+
+  def add(dish)
+    if no_dishes_available?
+      menu_number = 1
+    else
+      menu_number = dishes_available.keys.max + 1
+    end
+    @dishes_available[menu_number] = dish
+  end
+
+  def view
+    raise "No meal options have been added to this menu" if no_dishes_available?
+    dishes_available.each do |k, v|
+      puts "#{k}: #{v.name} | Price: £#{v.price}"
+    end
+  end
+
+  def select(order_number)
+    raise "No meal options have been added to this menu" if no_dishes_available?
+    raise "There are no dishes matching the number given" if !dishes_available.keys.include?(order_number)
```

Comment:
Metrics/LineLength: Line is too long. [106/80]<br>Style/NegatedIf: Favor `unless` over `if` for negative conditions.<br>Style/SignalException: Use `fail` instead of `raise` to signal exceptions.